cmd/notation/inspect.go

```diff
@@ -60,25 +60,25 @@ func inspectCommand(opts *inspectOpts) *cobra.Command {
 	if opts == nil {
 		opts = &inspectOpts{}
 	}
-	longMessage := `Inspect all signatures associated with the signed artifact.
+	longMessage := `Inspect all signatures associated with the signed artifact.`
+	exampleMessage := `# Inspect signatures on an OCI artifact identified by a digest:
+notation inspect <registry>/<repository>@<digest>
 
-Example - Inspect signatures on an OCI artifact identified by a digest:
-  notation inspect <registry>/<repository>@<digest>
+# Inspect signatures on an OCI artifact identified by a tag  (Notation will resolve tag to digest):
+notation inspect <registry>/<repository>:<tag>
 
-Example - Inspect signatures on an OCI artifact identified by a tag  (Notation will resolve tag to digest):
-  notation inspect <registry>/<repository>:<tag>
-
-Example - Inspect signatures on an OCI artifact identified by a digest and output as json:
-  notation inspect --output json <registry>/<repository>@<digest>
+# Inspect signatures on an OCI artifact identified by a digest and output as json:
+notation inspect --output json <registry>/<repository>@<digest>
 `
 	experimentalExamples := `
-Example - [Experimental] Inspect signatures on an OCI artifact identified by a digest using the Referrers API, if not supported (returns 404), fallback to the Referrers tag schema
-  notation inspect --allow-referrers-api <registry>/<repository>@<digest>
+# [Experimental] Inspect signatures on an OCI artifact identified by a digest using the Referrers API, if not supported (returns 404), fallback to the Referrers tag schema
+notation inspect --allow-referrers-api <registry>/<repository>@<digest>
 `
 	command := &cobra.Command{
 		Use:   "inspect [reference]",
 		Short: "Inspect all signatures associated with the signed artifact",
 		Long:  longMessage,
+		Example: exampleMessage + experimentalExamples,
```

Comment:
This seems not right as `experimentalExamples` should not be added if the experimental flag is not on.

Reply:
This also applies to other occurrences.

Reply:
fixed